### PR TITLE
Bump RuboCop to latest 0.90.0 version

### PR DESCRIPTION
**Pull request by peterberkenbosch**
## Summary

Bump Rubocop to 0.90.0

Related: https://github.com/rubocop-hq/rubocop/issues/8713
and https://github.com/solidusio-contrib/solidus_affirm_v2/commit/448b229b6b35f62e2ef8e41e944db221684d71a7

## Checklist

- [ ] I have structured the commits for clarity and conciseness.
- [ ] I have added relevant automated tests for this change.

### CODE REVIEW

**Comment by elia** [comment]
There are frequent IO/timeout errors on the CI lately, I restarted it, once it's green I think we can merge this PR

File: solidus_dev_support.gemspec [hunk 1]
```diff
@@ -40,7 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'puma', '~> 4.3'
   spec.add_dependency 'rspec_junit_formatter'
   spec.add_dependency 'rspec-rails', '~> 4.0.0.beta3'
-  spec.add_dependency 'rubocop', '~> 0.87.1'
+  spec.add_dependency 'rubocop', '~> 0.90.0'
```
Comment (ccarruitero) [inline solidus_dev_support.gemspec:43]: I think we should relax rubocop version, so you can update in your project if needed.

Something like:

```suggestion
  spec.add_dependency 'rubocop', '~> 0.90'
```

Btw, current rubocop version is `0.91` now https://rubygems.org/gems/rubocop

Comment (peterberkenbosch) [reply solidus_dev_support.gemspec]: @ccarruitero Thanks! I totally agree, updated that just now.

**Review by aldesantis — APPROVED** [review]

**Comment by aldesantis** [comment]
Merging even though the CI is failing — the timeouts are unrelated and have been resolved in master.

**Pull request merged**